plugin/build.gradle.kts

```diff
@@ -0,0 +1,144 @@
+import kotlinx.team.infra.*
+
+val jmhVersion: String by project
+val kotlin_version: String by project
+
+buildscript {
+    val kotlin_version: String by project
+    val infra_version: String by project
+    
+    val kotlinDevUrl by extra { rootProject.properties["kotlin_repo_url"] }
```

Comment:
Introducing this extra property doesn't seem to simplify accessing the project property. Should we use the `"kotlin_repo_url"` property directly?

Reply:
Yes, I have addressed this in the latest commit.